mg/deps.go

```diff
@@ -89,6 +92,9 @@ func runDeps(ctx context.Context, fns ...interface{}) {
 				}
 				wg.Done()
 			}()
+			if os.Getenv("MAGEFILE_VERBOSE") != "" {
```

Comment:
this is defined in the Verbose() method, so we can reuse that -  `if Verbose() {`